packages/cli/src/telemetry/send.js

```diff
@@ -0,0 +1,95 @@
+import path from 'path'
+
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http'
+import { Resource } from '@opentelemetry/resources'
+import fs from 'fs-extra'
+
+import { getPaths } from '@redwoodjs/project-config'
+
+import { getResources } from './resource'
+
+async function main() {
+  // Get all telemetry files
+  const telemetryDir = path.join(getPaths().generated.base, 'telemetry')
+  fs.ensureDirSync(telemetryDir)
+  const telemetryFiles = fs.readdirSync(
+    path.join(getPaths().generated.base, 'telemetry')
+  )
+
+  // Compute all the resource information
+  const customResourceData = await getResources()
+  const resource = Resource.default().merge(new Resource(customResourceData))
+
+  const traceExporter = new OTLPTraceExporter({
+    url:
+      process.env.REDWOOD_REDIRECT_TELEMETRY ||
+      'https://quark.quantumparticle.io/v1/traces',
+  })
+
+  // Go through all telemetry files and send the new spans to the telemetry collector
+  for (const [index, file] of telemetryFiles.entries()) {
+    // '_' denotes a file that has already been sent
+    if (file.startsWith('_')) {
+      continue
+    }
+
+    // Read the saved spans
+    const spans = JSON.parse(
+      fs.readFileSync(path.join(telemetryDir, file), { encoding: 'utf8' })
+    )
```

Comment:
You can save yourself some work here since you're using fs-extra

```suggestion
    const spans = fs.readJsonSync(path.join(telemetryDir, file))
```